monai/fl/utils/constants.py

```diff
@@ -51,20 +51,6 @@ class FlStatistics(StrEnum):
     FEATURE_NAMES = "feature_names"
 
 
-class RequiredBundleKeys(StrEnum):
-    BUNDLE_ROOT = "bundle_root"
```

Comment:
It seems the BUNDLE_ROOT key itself is not required in the configs anymore. However, I think we should keep `bundle_root` as an input argument to `MonaiAlgo` and then create the workflow objects there.

Reply:
Hi @holgerroth ,

Please take a look at all the required properties of a MONAI model-zoo bundle:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/bundle/properties.py#L31

Thanks.